### PR TITLE
feat: cart page

### LINKED ISSUE #4: Cart Page
### Cart Page
- [x] Horizontal products in cart
- [x] Cart details with bill summary

### PR DESCRIPTION
**Pull request by siddhigate**
### Cart page:
- Products list in cart
- Cart details

**Live preview:** https://fashionly-ui-git-cart-page-siddhigate.vercel.app/pages/cart.html

image

### CODE REVIEW

**Review by adidoshi — APPROVED** [review]
Looks great, good to go.👍
Just a small suggestion, if we can write comments for each section in the HTML file, for better readability.

File: css/base/utilities.css [hunk 1]
```diff
@@ -33,6 +33,15 @@
 }
 
 @media screen and (max-width: 700px) {
+
+  .flex-col-sm {
+    flex-direction: column;
+  }
+
+  .flex-items-start-sm {
```
Comment (msdeshmukh009) [inline css/base/utilities.css:45]: Just a question why is there `-sm` in the class name.  Maybe you can use these classes for the large screens as well.

**Comment by siddhigate** [comment]
> Looks great, good to go.👍 Just a small suggestion, if we can write comments for each section in the HTML file, for better readability.

Thanks for your suggestion, made the changes👍

File: css/pages/cart.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  font-size: 0.75rem;
+  padding: 0.25rem 0.5rem;
+}
+
+.billing-container > div {
+  padding: 0.5rem;
+}
+
+.address-container {
+  font-size: 0.9rem;
```
Comment (msdeshmukh009) [inline css/pages/cart.css]: Utility classes for font sizes would be a good idea cause I see you are repeating `font-size: 0.9em/1rem;` in multiple places

**Review by someshtiwarii — APPROVED** [review]
looks clean ! good to go

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good!!

File: css/base/utilities.css (hunk 1 above)
Comment (siddhigate) [reply css/base/utilities.css:45]: These are just the responsive variants. For responsive screens, I required flex-items-center for larger screens and flex-items-start for smaller screens. For that reason, I have made utility classes in the respective media queries. That's why I have used `class="flex-items-center flex-items-start-sm"`. 
This is inspired from Bootstrap. You may check out the responsive variants for flex grid in the Bootstrap docs.

Comment (msdeshmukh009) [reply css/base/utilities.css:45]: Ok Cool. I haven't used bootstrap that much maybe I will check it out. Thanks!!

File: css/pages/cart.css (hunk 2 above)
Comment (siddhigate) [reply css/pages/cart.css]: Thanks for pointing this out! Made the changes accordingly

**Pull request merged**